PhysicsTools/PatAlgos/python/producersHeavyIons/heavyIonJets_cff.py

```diff
@@ -70,6 +70,14 @@
     patJetCorrFactors
 )
 
+from Configuration.ProcessModifiers.run2_miniAOD_pp_on_AA_103X_cff import run2_miniAOD_pp_on_AA_103X
```

Comment:
This is from #33928: a rebase could get rid of these spurious additions in the diff

Reply:
Thanks for spotting, I made some mistakes when moving around directories.